src/tests/integration/test_postgreSQL.py

```diff
@@ -42,3 +42,31 @@ def test_pgspecial(ip_with_postgreSQL):
     out = ip_with_postgreSQL.run_cell("%sql \l").result  # noqa: W605
 
     assert "postgres" in out.dict()["Name"]
+
+
+@pytest.mark.parametrize(
+    "query, expected",
+    [
+        (
+            "%sql select '{\"a\": 1}'::jsonb -> 'a';",
```

Comment:
Please add cases for 
1. `json`
2. multiline

Reply:
Added cases for [json](https://github.com/ploomber/jupysql/pull/938/files#diff-d88bd9d85509a4c9360ff35ecb4ca1f4ebbc2c2cdbde1b00de1c203d0d3af2e1R66-R81) and [multiline/whitespace](https://github.com/ploomber/jupysql/pull/938/files#diff-d88bd9d85509a4c9360ff35ecb4ca1f4ebbc2c2cdbde1b00de1c203d0d3af2e1R82-R111)